lib/active_model_serializers/test/schema.rb

```diff
@@ -60,11 +60,11 @@ def call
         attr_reader :document_store
 
         def controller_path
-          request.filtered_parameters[:controller]
+          request.filtered_parameters.with_indifferent_access[:controller]
```

Comment:
See https://github.com/rails-api/active_model_serializers/pull/1947#pullrequestreview-5497390

I couldn't write a failing test (in the existing suite), but there is a sample app with a test failing.